### PR TITLE
Sen 94 & 97

### PR DESCRIPTION
**Pull request by Fungucide**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -141,7 +143,7 @@ app.post("/driver/updateUserAccount", async (request, response) => {
 
 app.post("/login/createAccount", async (request, response) => {
     try {
-        var res = await loginService.createAcount(request.query.username, request.query.password, request.query.type);
+        var res = await loginService.createAcount(request.body.username, request.body.password, request.body.type);
```
Comment (NElnour) [inline server/index.js]: can you check with Jonathan if we're still using async/await?

File: server/index.js [hunk 2]
```diff
@@ -155,7 +157,7 @@ app.post("/login/createAccount", async (request, response) => {
 
 app.post("/login/authenticate", async (request, response) => {
     try {
-        var res = await loginService.authenticate(request.query.username, request.query.password);
+        var res = await loginService.authenticate(request.body.username, request.body.password);
```
Comment (NElnour) [inline server/index.js]: same as above: check with Jonathan if we're still using async/await

Comment (Fungucide) [reply server/index.js]: He said that async/await is fine.

**Comment by JonathanYcWang** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- server/Services/LoginService.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CSC492-DSC-Community-Projects/Backend/pullRequest?prid=5141321&bid=16889383)

**Comment by JonathanYcWang** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- server/Services/LoginService.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CSC492-DSC-Community-Projects/Backend/pullRequest?prid=5141321&bid=16889383)

File: server/Services/LoginService.js [hunk 3]
```diff
@@ -1,37 +1,65 @@
+const TYPES = { "GroceryStores": "Grocery Store", "FoodBank": "Food Bank", "Drivers": "Driver" };
```
Comment (NElnour) [inline server/Services/LoginService.js]: can we make food bank key consistent with others (as plural)?

Comment (Fungucide) [reply server/Services/LoginService.js]: Yeah I'll rename the collection and update the code.

**Comment by JonathanYcWang** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- server/Services/LoginService.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CSC492-DSC-Community-Projects/Backend/pullRequest?prid=5141321&bid=16889383)

**Pull request merged**